Sources/ClientRuntime/Networking/Http/HttpBody.swift

```diff
@@ -14,29 +14,20 @@ public extension HttpBody {
     static var empty: HttpBody {
         .data(nil)
     }
-}
-
-extension HttpBody {
```

Comment:
iirc i needed public access to `toBytes` but maybe not. I'll double check and update accordingly

Reply:
Confirmed we need to access `toBytes` outside of this module. 

We no longer need to the `toAwsInputStream` because CRT changed how input streams are defined. Now its a protocol, `IStreamable` that `ByteBuffer` conforms to.